lib/standard_deviation.rb

```diff
@@ -1,5 +1,18 @@
 ages = [24, 30, 18, 20, 41]
 
-# Your code here for calculating the standard deviation
-
-# When you find the standard deviation, print it out
+def standard_deviation(ages)
```

Comment:
This algorithm looks awesome, but I would have liked to see this in some sort of class - whether you move it into the Event class, or create a Calculations class with this standard deviation method.

Also, this method is doing *many* things - the standard deviation algorithm has multiple steps and we would like to see those steps broken out into multiple methods.